src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -35,24 +32,51 @@ import * as SessionUtils from '../../SessionUtils';
 import NotFoundPage from '../../../pages/ErrorPage/NotFoundPage';
 import getRootNavigatorScreenOptions from './getRootNavigatorScreenOptions';
 import DemoSetupPage from '../../../pages/DemoSetupPage';
+import * as OnyxTypes from '../../../types/onyx';
```

Comment:
I think in other migrations we were importing OnyxTypes individually so I would do it also here for consistency

Reply:
@kubabutkiewicz Otherwise, there will be duplicates in import naming, Session type, and Session lib for example. So in cases like this, we import all as OnyxTypes.
![image](https://github.com/Expensify/App/assets/23176449/21bd411c-9421-4167-83d3-47315635d152)